prost-build/src/lib.rs

```diff
@@ -658,6 +659,17 @@ impl Config {
         self
     }
 
+    /// Configures what filename protobufs with no package definition are written to.
+    ///
+    /// If unset protobuf compilation will fail if there are any proto files without a package definition.
```

Comment:
Is there a specific error we can call out here for when this case happens?

Reply:
I've removed the comment, since that isn't actually the case. It's optional to call this function and if it isn't called and there are protos without a package spec it won't fail.